l3experimental/l3opacity/build.lua

```diff
@@ -10,5 +10,23 @@ module = "l3opacity"
 -- Location of main directory: use Unix-style path separators
 maindir = "../.."
 
+-- Set up to allow testing dvips, etc.
+specialformats = specialformats or {}
+specialformats.latex =
+  {
+    luatex = {binary = "luahbtex",format = "lualatex"},
+    ptex  = {binary = "eptex"},
+    uptex = {binary = "euptex"},
+    ["etex-dvips"] = {binary = "etex", format = "latex"},
+    ["etex-dvisvgm"] =
+      {
+        binary = "etex",
+        format = "latex",
+        tokens = "\\ExplSyntaxOn\\sys_load_backend:n{dvisvgm}\\ExplSyntaxOff"
+      }
+  }
```

Comment:
Yes, I think that would make sense: eventually we will likely want them for `l3kernel` anyway.

Reply:
Can be done in a following PR.

Also I'm not sure if `luatex` engine can use `luahbtex` unconditionally.
```
luatex = {binary = "luahbtex",format = "lualatex"}
```

Reply:
Release LuaLaTeX is always LuaHBTeX, so yes :)

Reply:
I think the answer is yes, since `l3build` has been using `luahbtex` for `specialformat.latex.luatex` for 2019 or later of `texlua` since middle 2020 (see https://github.com/latex3/l3build/commit/5cd9d6981fa40d07fc8e2ec2bda4293c8f744c6a).

https://github.com/latex3/l3build/blob/f976c438fb74964e5376935dce1efdd8c821b911/l3build-variables.lua#L132-L137
```lua
if not string.find(status.banner,"2019") then
  specialformats.latex.luatex = specialformats.latex.luatex or
    {binary = "luahbtex",format = "lualatex"}
  specialformats["latex-dev"] = specialformats["latex-dev"] or
    {luatex = {binary="luahbtex",format = "lualatex-dev"}}
end
```

So the custom `specialformats` setting in this repo can also be simplified a bit.

Reply:
I'll come back to `specialformat` stuff after completing two follow-up `l3opacity` PRs, one for testing with pdfmanagement and one for speeding up a bit.

Is it permissible to change back to `checksearch = true` for `l3opacity`, to load pdfmanagement support files from system `texmf`? Currently `checksearch` is already turned on in several configs.

```
$ rg --no-heading 'checksearch\s*=\s*true'
l3kernel/config-l3doc.lua:3:checksearch  = true
l3trial/l3str-format-new/build.lua:14:checksearch = true
l3trial/l3htoks/build.lua:16:checksearch  = true
l3trial/l3doc/build.lua:19:checksearch  = true
l3trial/l3auxdata/build.lua:15:checksearch = true
```

Reply:
Yes, we can do that

Reply:
> I'll come back to `specialformat` stuff ... .

See #1482.